### DISCUSSION TITLE
Has anyone ported this to Apple's M1 (or above) chips yet?

### DISCUSSION THREAD

**Discussion by miscvoid** [cia-foundation/TempleOS#20] [Ideas]
Really think this might be a good fit for a macbook pro if anyone has done it already

**Comment by sieadev** [comment]
I could imagine this to be rather complicated. Why do you wanna run anything but the in-house OS on a mac anyways?

**Comment by WeAreBlank16** [comment]
It might be easier to make your own custom touchscreen device instead of overwriting apple's co-dependant kernel-hardware design.

**Reply by WeAreBlank16** [reply]
Well, I looked more into it, and if your iPad's specs are good, you might be able to run the Virtual Machine app "UTM", and that way you could technically run a different OS without it getting too complicated.
